var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -20,7 +20,7 @@ class Paraview(CMakePackage, CudaPackage):
     maintainers = ['chuckatkins', 'danlipsa', 'vicentebolea']
 
     version('master', branch='master', submodules=True)
-    version('5.9.1-RC1', sha256='bb3f12bd3d31bcb52455e1393e45f81a18754d3c3e8199d51532b0c067dc1595')
+    version('5.9.1-RC2', sha256='87395d7d65e96d2a5f8a8086b5da52a78c7929397a03170cbd04c1e3b3bb501c')
```

Comment:
Confirmed sha256

Reply:
Is there a reason for dropping `RC1`?  It *is* still available on the site so someone *could* try to build it.

Reply:
I do not feel strong about this,if it can be useful to anybody I do not mind to keep the RCs until the final release is done, then we can get rid of all at once